mm2src/mm2_net/src/sse_handler.rs

```diff
@@ -14,51 +14,49 @@ pub async fn handle_sse(request: Request<Body>, ctx_h: u32) -> Result<Response<B
         Err(err) => return handle_internal_error(err).await,
     };
 
-    let config = match &ctx.event_stream_configuration {
-        Some(config) => config,
-        None => {
-            return handle_internal_error(
-                "Event stream configuration couldn't be found. This should never happen.".to_string(),
-            )
-            .await
-        },
+    let access_control = ctx.conf["access_control_allow_origin"]
```

Comment:
IMO we should bring back the event stream configuration field and keep these fields in there. We have `rpccors` on the same level with `access_control_allow_origin` which makes confusions.

Reply:
in https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172#discussion_r1788537377